improver/synthetic_data/set_up_test_cubes.py

```diff
@@ -261,6 +348,10 @@ def set_up_variable_cube(
             Grid resolution (degrees for latlon or metres for equalarea).
         domain_corner (Optional[Tuple[float, float]]):
             Bottom left corner of grid domain (y,x) (degrees for latlon or metres for equalarea).
+        height_levels (Optional[List[float]]):
+            List of altitude/pressure levels.
+        pressure (Optional[bool]):
+            Flag to indicate whether to make height levels pressure levels.
```

Comment:
How about `Flag to indicate whether the height levels are specified as pressure, in Pa.  If False, use height in metres.`

And above I might specify the units (height in metres or pressure in Pa).

Reply:
Updated [here](https://github.com/metoppv/improver/commit/1d7f21003ca70fbe79b8c8943e280bc37f411b98#diff-ea38258437543d699495d248fce61e29L351).